orbit/README.md

```diff
@@ -180,6 +180,13 @@ Press Enter to continue, or Control-c to exit.
 [...]
 ```
 
+If you want to run orbit from source directly, you can do the following:
+
+```sh 
+cd orbit
```

Comment:
Do you need to `cd` into if you are executing `go run` with the full package name?